metrics/metrics.go

```diff
@@ -19,16 +19,36 @@ const (
 var (
 	meter otelapi.Meter
 
+	GetHeaderLatencyHistogram    otelapi.Float64Histogram
 	GetPayloadLatencyHistogram   otelapi.Float64Histogram
 	PublishBlockLatencyHistogram otelapi.Float64Histogram
 
+	GetHeaderCount  otelapi.Int64Counter
+	GetPayloadCount otelapi.Int64Counter
```

Comment:
**suggestion:**

if there is a histogram for this type of even, then no need to also create a counter because the histogram will automatically create the total count as one of the metrics.